app/controllers/gitlab/projects/events_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Gitlab
+  module Projects
+    class EventsController < WebhookBaseController
+      def create
+        Gitlab::ProjectEventsService.perform(params)
```

Comment:
The api doesn't expect any response, the documentation states that they expect any response, otherwise they assume the data didn't arrive and send it again, the `head :ok` is meant to handle that. But there is no error handling on this because there is no way to do anything if the API sends us an bad request, we just trow an error and nothing happens.